llama_index/readers/web.py

```diff
@@ -23,6 +27,11 @@ class SimpleWebPageReader(BaseReader):
             Default is None.
     """
 
+    is_remote: bool = True
+    html_to_text: bool
+
+    _metadata_fn: Optional[Callable[[str], Dict]] = PrivateAttr()
```

Comment:
Double checking: right now serializing/deserializing would forget any custom metadata fn

Reply:
Yea, some loaders have configs that aren't serializable

The beautiful soup loader is another example